x-pack/test/functional/services/ml/data_visualizer_file_based.ts

```diff
@@ -132,6 +132,17 @@ export function MachineLearningDataVisualizerFileBasedProvider(
       });
     },
 
+    async assertIngestedDocCount(count: number) {
+      const docCount = await mlCommonUI.getEuiDescriptionListDescriptionFromTitle(
+        'dataVisualizerFileImportSuccessCallout',
+        'Documents ingested'
+      );
+      expect(docCount).to.eql(
+        count,
+        `Expected Documents ingested name to be '${count}' (got '${docCount}')`
```

Comment:
IMO the `name` here could be a bit confusing. Wouldn't it be clearer to have `count` or `value` instead?

Reply:
Fixed in https://github.com/elastic/kibana/pull/109274/commits/936a18d22787a1d5e2916e3e1c45519ca8834f99